### PR TITLE
Fix for fair component search 📱

### PR DESCRIPTION
**Pull request by ashfurrow**
This PR fixes LD-207. It's a WIP because of the TODOs left in the code.

### CODE REVIEW

**Comment by ashfurrow** [comment]
I'm going to test on-device still, but this is ready for review.

File: Artsy/View_Controllers/Util/ARNavigationController.m [hunk 1]
```diff
@@ -150,7 +150,7 @@ - (UIInterfaceOrientation)preferredInterfaceOrientationForPresentation
     return self.topViewController.preferredInterfaceOrientationForPresentation ?: UIInterfaceOrientationPortrait;
 }
 
-#pragma mark - UINavigationControllerDelegate
+#pragma mark - po
```
Comment (orta) [inline Artsy/View_Controllers/Util/ARNavigationController.m]: ^

Comment (ashfurrow) [reply Artsy/View_Controllers/Util/ARNavigationController.m]: Haha, classic.

**Comment by ashfurrow** [comment]
@orta any further feedback on this, or is it good to merge?

**Comment by orta** [comment]
Nope - sorry @l2succes you'll probably get merge conflicts

**Pull request merged**